packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -56,25 +53,29 @@ void main(List<String> args) async {
 
     for (final testFile in testFiles) {
       final testTarget = testFile.path;
-      _maybeAddOfflineArgument(modifiableArgs, testTarget);
-      _maybeAddExperimentsArgument(modifiableArgs, testTarget);
-      final testRunnerArgs = TestArgs([
-        ...modifiableArgs,
-        '${TestArgs.testTargetArg}$testTarget',
-      ]);
-      await runFlutterIntegrationTest(testRunnerArgs);
+      modifiableArgs.add('${TestArgs.testTargetArg}$testTarget');
+      await _runTest(modifiableArgs, testTarget);
     }
   }
 }
 
-void _maybeAddOfflineArgument(List<String> args, String testTarget) {
-  if (testTarget.startsWith(_offlineIndicator)) {
-    args.add(TestArgs.offlineArg);
+Future<void> _runTest(List<String> modifiableArgs, String testFilePath) async {
+  _maybeAddOfflineArgument(modifiableArgs, testFilePath);
```

Comment:
is this still necessary? It looks like we are no longer using `TestArgs.offline`

Reply:
We stopped using experiments, but still using offline. I think it is convenient to have them in separate folders and not having to provide additional signal that they are offline.

Reply:
It looks like we don't use the `TestArgs.offline` field anymore though. We are now using the absence of a test app uri to signal "offline" right?

Reply:
True. Cleaned up both offline and experiments TestRunnerArgs.